packages/cmf/__tests__/bootstrap.test.js

```diff
@@ -0,0 +1,180 @@
+import { render } from 'react-dom';
+import { routerMiddleware, syncHistoryWithStore } from 'react-router-redux';
+import createSagaMiddleware from 'redux-saga';
+
+import bootstrap, * as internals from '../src/bootstrap';
+import { registerInternals } from '../src/register';
+import actionCreator from '../src/actionCreator';
+import actions from '../src/actions';
+import { assertTypeOf } from '../src/assert';
+import component from '../src/component';
+import expression from '../src/expression';
+import storeAPI from '../src/store';
+import sagaRouter from '../src/sagaRouter';
+import sagas from '../src/sagas';
+
+jest.mock('react-dom', () => ({
+	render: jest.fn(),
+}));
+jest.mock('react-router-redux', () => ({
+	routerMiddleware: jest.fn(() => ({ routerMiddlewareMocked: true })),
+	syncHistoryWithStore: jest.fn((history, store) => ({ mockedSync: true, history, store })),
+}));
+jest.mock('redux-saga', () => {
+	const run = jest.fn();
+	const middleware = jest.fn(() => ({ reduxSagaMocked: true, run }));
+	middleware.run = run;
+	middleware.clearRun = () => run.mockClear();
+	return middleware;
+});
+
+// we mock all internal dependencies
+jest.mock('../src/actionCreator', () => ({
+	registerMany: jest.fn(),
+}));
+jest.mock('../src/component', () => ({
+	registerMany: jest.fn(),
+}));
+jest.mock('../src/expression', () => ({
+	registerMany: jest.fn(),
+}));
+jest.mock('../src/sagas', () => ({
+	registerMany: jest.fn(),
+}));
+jest.mock('../src/register', () => ({
+	registerInternals: jest.fn(),
+}));
+
+jest.mock('../src/store', () => ({
+	addPreReducer: jest.fn(),
+	setHttpMiddleware: jest.fn(),
+	setRouterMiddleware: jest.fn(),
+	initialize: jest.fn(() => ({ dispatch: jest.fn(), applyMiddleware: jest.fn(), })),
+}));
+
+describe('bootstrap', () => {
+	describe('bootstrap registry', () => {
```

Comment:
It is already in a describe containing `bootstrap`. No need to repeat this word or in the description you will have `bootstrap bootstrap registry`.
Same for next describes